Cargo.toml

```diff
@@ -52,7 +52,7 @@ serde_json = "1.0"
 tokio = { version = "0.2.2", features = ["fs", "macros", "io-std", "rt-util", "sync", "time", "test-util"] }
 tokio-test = "0.2"
 tower-util = "0.3"
-url = "1.0"
+url = "2.0"
```

Comment:
I'm by no means the maintainer of hyper, but I would imagine keeping this separate would be best.